airbytehq/airbyte
# Bump Airbyte version from 0.40.3 to 0.40.4

*IMPORTANT: Only merge if the platform build is passing!*

Changelog:

6418b32661 🪟 🔧 🧹 Migrate attempt `bytesSynced` to `totalStats.bytesEmitted` and cleanup `AttemptDetails` component (#16126)
c91385deba Improve behavior of password input field (#16011)
a43c098acb Improve airbyte-metrics support in the Helm chart (#16166)
a9a1b61ef4 fix: Remove Fullstory leftovers (#16223)
31215cb5c8 Destination MongoDB: Remove additionalProperties:false (#16025)
9acc802c71 🎉 MySQL sources: emit state messages more frequently (#16259)
fc13413c9b Enforce secure connection in destination-postgres-strict-encrypt (#16260)
5ad492c074 🎉 MS SQL sources: emit state messages more frequently (#16261)
db8df934b1 [13309] Destination-mssql: enabled DAT tests (#16226)
29cef6635f 🎉 Redshift & snowflake sources: emit state messages more frequently (#16258)
c3f3702522 🎉 Jdbc sources: emit state messages more frequently (#16256)
ed8a04dafc Bump com.bmuschko:gradle-docker-plugin:8.0.0 (#16255)
dceeef4683 [cdk] pin dataclasses-jsonschema to 2.15.1 (#16253)
fd547c64d7 🐛 Avro related destinations: fix field name clashing with combined restrict (#16243)
a9a833209c Add protocol_version to e2e testing connectors (#16198)
29ec2617ec Add missing pr id
c2c276d1a4 Add documentation on how to configure and debug connector resources. (#16204)
fbac1d3fa5 🎉 Oracle source: emit state messages more frequently (#16238)
6319c2acff Update Google Analytics GA doc (#16190)
be56011abd airbyte-config: Implement RemoteDefinitionsProvider (#16018)
9a883adc6a 🐛 S3 destination: fix json to avro converter when there is field name clashes (#14492)
4f03c09d48 Add documentation for running the dev image of a connector (#16176)
c9326170ab doc(fix): update minor typo on helm deploy (#16193)
f9b20f9995 Add icon for Destination-TiDB (#16207)
570ceaed29 docs: updated collection metrics doc (#16227)
d83abd6672 Remove foreign key constraint for actor_definition_id in actor_oauth_param (#16118)
bedccd6146 Source Plaid: minimum last updated datetime support added for Capital One items (#15231)
36ac7836a3 🪟 🐛 Remove Debug Info button from UI pending API update (#16188)
235cbdf09b add CombinedConnectorCatalog (#16203)
21ddc93731 chore: cleanup actions and workflows (#16139)
a03509bd5c 🪟 🔧 Remove Fullstory script (#16217)
7efe798b41 🐛 Source Amplitude: Re-release on new `airbyte_cdk==0.1.81` (#16185)
8737aedd4a 🎉 Source Pinterest: increase `unit_test` coverage up to 90% (#16164)
5773c224bc Destination Redshift: custom JDBC parameters (#16208)
3611e656c7 🐛 Source Tiktok: unnest `cursor` and `primary key` to the root-level (#16137)
a475235d89 📝 S3 source: update doc about path_prefix + path_pattern (#16206)
35f4c6edf8 Docker Operational Improvements. (#16195)
827e7a55ac Add protocol version migration (#16145)
944b168b2e Fix error finding PostgreSQLContainer in migration tasks (#16146)
4cec0bae8f Auth Button now shows re-authenticate when it has been authenticated once (#15615)
a6b3d1f43e Ensure scheduleType gets updated when connection is updated (#16183)
e45709a817 Source Snowflake: Docs Fix invalid config examples (#16158)
d4524032ae 🎉 New Destination: TiDB (#15592)
3cfa489234 CDK: Fix regression in `_checkpoint_state` arg (#16141)
8998bad8c7 Updated connector catalog page (#16181)
86a5f90f2e Edited July Release Notes (#16179)
c6f876d618 enforce ssl in strict mysql and postgres source (#16114)
b9bba17153 Fixed release name
bef126939b Update august_2022.md
732ec3ca81 Release Note Documentation (#16148)
cd2860f617 chore: cleanup dormant workflows (#16022)
e560ed7bfe Handle SQLSTATE Exception (08006) when reading records (#15869)
3ed7f75032 🐛 Source Facebook Marketing: remove "end_date" from config if empty value (#16096)
2967dfaaaa Revert "Use a valid pat for slash command dispatch (#16144)" (#16156)
4892a08033 Frontend: update AttemptDetails.tsx (#15795)
aab03770c9 🐛 Source SalesForce: better detect API type (#16086)
20a5b9834a :window: :tada: Connection job list pagination (#15938)
a4628916b4 Use a valid pat for slash command dispatch (#16144)
184f192bca Revert Sendgrid and Sentry sources to use python CDK instead of low-code (#16112)
0910dd145b 🐞 Postgres source: remove config logging (#16138)
4e48dad93b :window: :art: Add tooltip to source-defined cursors and primary keys (#16116)
56d19bbc4d Edit Managing Airbyte Cloud doc (#16111)
009f0eb93c Edit metrics formatting (#16105)
0f0e6606d3 Turn on OAuth logins by default (#16122)
ce2bcd4207 fix: run.delete() not available in pygithub (#16127)
bde0abcfd5 Source TypeForm: improve `metadata.referer` url parsing (#16125)
e6c6ecfc14 Docs: Fix Typo in Schema.md (#16121)
ff7cdcc345 🪟 Use source defined primary keys over configured (#16107)
7de1e522dd Update all acceptance tests with old postgres versions (#16120)
4016c1473d 🪟 🧹 Use scheduleData instead of schedule for connection (#15924)
70136d3060 Source Redshift: added custom JDBC parameters (#16058)
ab98441488 Source Salesforce: exclude KnowledgeArticleVersion from bulk api (#16119)
d9089ed2ce Add is_test db config to test harness (#16117)
6ad5904be6 Source Bing Ads: Fix unreadable log messages (#15987)
4fbe03bebe Add explicit tracking of Airbyte Protocol Version in ConnectorSpecification (#15340)
d6ae694367 🎉Source Twilio: Added state checkpoint interval (#16110)
d1f44cc0f5 Revert "Refactor test harness in preparation for cloud acceptance tests (#15932)"
05fdda7d7f Change abnormal jobs metrics report from reporting number of connections to percentage of connections (#15740)
aceaf28335 feat: disable ssh tunnel idle timeout (#15635)
d333abfcaa Bumped release stage to beta (#16102)
97c28eb40c :window: :bug: Prevent flickering when resizing documentation split view to specific size (#15996)
0f045d652e Fix mismatch between container when setting limits (#16047)
6074ad5d83 feat: tools to identify and clean up unused workflows (#16019)
78ca6c7f84 🪟🧹 Stylelint (#15452)
f944266a83 🪟 Show Validation Errors on Service Form for Missing Fields (#15625)
dae056c552 Refactor test harness in preparation for cloud acceptance tests (#15932)
c98ed0fb44 Source Twilio: increase unit_test cov up to 90% (#16084)
d999ba1568 Source Twilio: shift start date for stream if it exceeds 400 days (#15972)
4c8739c62c Added details on Refresh Token for Slack (#15995)
749b22fbf4 Fix source or destination status from disappearing and prevent discard changes confirmation modal after delete (#15889)
d7448f4f75 Source slack: unit tests (#16085)
052563771f 🪟 Add "refresh schema" button back to create page (#15651)
bfc8249805 Source slack: fix integration tests (#16050)
1113ec0325 Source Recharge: Handle total_weight as a number (#16045)
a51a53fc3d New ssl flow for connectors (#15993)
6a06c5b0ff Upgrade OpenAPI Generator to 6.1.0-SNAPSHOT (#16026)
f3f895d6a9 Use docker-compose-cloud.buildx file (#15994)

Instructions:
- *SQUASH MERGE* this PR - this is necessary to ensure the automated Create Release action is triggered.
- Double check that the Create Release action was triggered and ran successfully on the commit to master (this should only take a few seconds).
- If the Create Release action failed due to a transient issue, retry the action. If it failed due to a non-transient issue, you will need to manually create a release by following these steps:
 1. Pull most recent version of master
 2. Run ./tools/bin/tag_version.sh
 3. Create a GitHub release with the changelog
